fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1157,6 +1187,167 @@ long getTotalRecordsScanned() {
         return totalRecordsScanned.get();
     }
 
+    /**
+     * A class holder for the config parameters needed to rebuild an online index.
```

Comment:
```suggestion
     * A holder for the config parameters needed to rebuild an online index.
```

I'm also not sure that "configuration paramers" is exactly right here. At the very most, it's something like "mutable configuration parameters"--i.e., the configuration parameters that one might want to update in the middle of an index building. (For example, the index is itself arguably a configuration parameter, but *not* one that would want to update while building it. Another parmater that doesn't make sense to change: the store's key space path.) These are mostly the kinds of parameters one would want in a hypothetical `ThrottlingRunner` except maybe `progressLogIntervalMillis` (though maybe that, too, should be in a throttling runner). Ideally, the description here would reflect that, but maybe just mentioning that these parameters are designed to be safe to be updated while a build is running is sufficient.

Reply:
sure, added some context.